inc/saimpls.h

```diff
@@ -81,6 +81,31 @@ typedef enum _sai_inseg_entry_attr_t
      */
     SAI_INSEG_ENTRY_ATTR_NEXT_HOP_ID,
 
+    /**
+     * @brief Use TC for a label.
+     *
+     * Associate TC using SAI_SWITCH_ATTR_QOS_EXP_TO_TC_MAP or SAI_PORT_ATTR_QOS_EXP_TO_TC_MAP.
+     * Mainly used for E-LSP kind of MPLS tunnels, where EXP infers both TC and COLOR
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default true
```

Comment:
is this backward compatible? should true to be default behavior or false?

Reply:
There was no QoS mapping in existing SAI spec for MPLS traffic. 
Default behavior proposal is to use "QoS map", i think comment is wrong here (Use TC for a label should be Use TC from a QoS Map).